File: 프로그래머스/2/12924. 숫자의 표현/숫자의 표현.java
class Solution {
    public int solution(int n) {
        int answer = 1;
        for(int i = 1; i <= n/2; i++) {
            int temp = i;
            for (int j = i + 1; temp < n; j++) {
                temp += j;
                if (temp == n) {
                    answer += 1;
                    break;
                }
            }
        }

        return answer;
    }
}

File: 프로그래머스/2/12924. 숫자의 표현/숫자의 표현.py
def solution(n):
    answer = 0
    start = 1
    while True :
        total = 0
        if start > 10000 :
            break
        for i in range(start, 10001) :
            total += i
            if total == n :
                answer += 1
            elif total > n :
                break
        start += 1
        
    return answer